### DISCUSSION TITLE
Translation error - French

### DISCUSSION THREAD

**Discussion by wilhelmbismarck** [ScratchAddons/ScratchAddons#7565] [Translation] [answered]
Hi - don't know if this is the correct place to report a translation error but I do it.

**-- Path**
in Scratch Forum, with [code][/code], SA allows to copy the code content.
**-- Error**
In french, the button "_copy code_" translation is « _Copi**é** le code_ », which is a really big grammatical error ("_copi**ed** the code_").
The translation should be « _Copi**er** le code_ ».

If there's a way of correcting typing myself, please let me know with an answer ^^

Have a good day !

**Comment by WorldLanguages** [comment, marked as answer]
Okay, I changed it
